custom_components/localtuya/config_flow.py

```diff
@@ -39,9 +39,9 @@
 BASIC_INFO_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_FRIENDLY_NAME): str,
+        vol.Required(CONF_LOCAL_KEY): str,
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_DEVICE_ID): str,
-        vol.Required(CONF_LOCAL_KEY): str,
```

Comment:
Doesn't matter that much, but was this necessary?

Reply:
@postlund Well, it's just a matter of speed if you need to configure many devices (and of my eternal laziness 😄 ), since the only param to be input after the name is the key, it just saves a couple of "tabs" during configuration. I am lazy, I now 😆

Reply:
Yeah, ok, it's not very pretty in the UI as it appears out-of-context 😉

Reply:
I partially agree, actually in my mind the first two params should be the only changeable (name and key) and the others should be disabled, so it's logically divided into two blocks of params...